tests/_support/Page/User.php

```diff
@@ -335,4 +335,19 @@ public function isRejectTheNewUser()
         $I->click('th.sorting_asc');
         $I->click("//td[text()='{$username}']/..//a[text()='Reject']");
     }
+
+    /**
+     * @Then /^I see last created user account display top of user list page$/
+     */
+    public function getLastCreatedAccountTopList()
+    {
+        $I = $this->I;
+
+        $username = $I->getRememberedString('lastNewUsername');
+        $I->amOnPage('/admin/user');
+        $I->see($username);
```

Comment:
The `Then` statement seems to indicate that this should be checking that the user is listed as the top line in the table, not just anywhere.

Reply:
@adlearningmate I think you worked on this, can you add the missing part to test if the pending user is actually at the top of the table?